src/main/resources/i18n/messages.properties

```diff
@@ -43,6 +43,7 @@ email.notification.group.editors=Editors
 # Notification Titles (based on originating type)
 email.notification.title.attachment=The attachment "{0}" for the lecture "{1}" in the course "{2}" was changed.
 email.notification.title.file.submission.successful=Your submission for the file upload exercise "{0}" in the course "{1}" was successful.
+email.notification.title.duplicate.test.cases="{0}" in the course "{1}" has multiple test cases with the same name! This critical issue needs to be resolved as quickly as possible. Otherwise, there will be problems with creating the student results!
```

Comment:
The exercises are often prepared a few weeks in advance, so it‘s not that urgent to fix the problem immediately. I don’t know if you should adapt this text a bit or even use a distinction between releases and unreleased exercises

Reply:
I explicitly formulated this because @krusche told me to include a text that should urge the instructor/editor to fix the issue as quickly as possible